LayoutTests/fast/rendering/render-style-null-optgroup-crash.html

```diff
@@ -0,0 +1,22 @@
+<style>
```

Comment:
Can we simplify this testcase at all?

Reply:
This is already the reduced version - Removing stuff any further from this doesn't reproduce the issue.